### DISCUSSION TITLE
Github projects?

### DISCUSSION THREAD

**Discussion by ryanheath** [ravendb/ravendb#12307] [Ideas]
Is it possible to add projects to Ravendb gtihub repo? See like https://github.com/twbs/bootstrap/projects

This way it is easy to track (or have a better sense) when releases are about to come out.
Also we can see what has gone into a release or patch to determine if an upgrade is worth it now.

// Ryan

**Comment by ayende** [comment]
Are actual bug tracking is done on https://issues.hibernatingrhinos.com/

You can see the changes going to 5.2 release, for example, here:

https://issues.hibernatingrhinos.com/issues/RavenDB?q=%235.2%20

**Comment by mcrio** [comment]
Hi, you can also look up the what's new page in the docs:
https://ravendb.net/docs/article-page/5.1/csharp/start/whats-new
